UK/Data/Settings/Lists_SMR.txt

```diff
@@ -95,6 +95,7 @@ m_Column:NFREQ:7:0:107:0:0:0:UK Controller Plugin:::1:0.0
 m_Column:RLS:3:1:124:9012:9015:1:UK Controller Plugin:UK Controller Plugin:UK Controller Plugin:1:0.0
 m_Column::6:0:125:9014:9014:1:UK Controller Plugin:UK Controller Plugin:UK Controller Plugin:1:0.0
 m_Column:PRE:3:1:127:9017:9018:0:UK Controller Plugin:UK Controller Plugin:UK Controller Plugin:1:0.0
+m_Column:QSY:7:0:107:0:0:0:UK Controller Plugin:::1:0.0
```

Comment:
Agree with this, is it hidden by default? Screenshot would help.

Reply:
Indeed hidden by default
![image](https://github.com/user-attachments/assets/c2d11d23-2ebf-479e-9a38-6f84c3c68208)